ietf/templates/nomcom/nomcom_private_base.html

```diff
@@ -132,7 +132,12 @@ <h1>
                     </li>
                     <li class="dropdown-item">
                         <a href="{% url 'ietf.nomcom.views.private_volunteers' year %}">
-                            View volunteers
+                            View volunteers (full)
+                        </a>
+                    </li>
+                    <li class="dropdown-item">
+                        <a href="{% url 'ietf.nomcom.views.private_volunteers' year "simple" %}">
```

Comment:
This is not doing what you intend, I think. There's nothing in the urls to reverse this to. If possible, lets talk about this in yokohama (I'm at the hackathon today).